code/src/index.css

```diff
@@ -1,13 +1,189 @@
+/** @format */
+
 body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+	margin: 0;
+	box-sizing: border-box;
+	font-family: "Roboto", sans-serif;
+	background-color: black;
+}
+
+.movie-list-container {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+}
+
+.movie-list-container a {
+	display: flex;
+	text-decoration: none;
+	color: white;
+}
+
+.movie-thumb-link {
+	position: relative;
+	text-decoration: none;
+}
+
+.thumb-img {
+	width: 100%;
+	height: auto;
+}
+
+.movie-thumb-text {
+	display: flex;
+	flex-direction: column;
+	justify-content: flex-end;
+	position: absolute;
+	bottom: 0;
+	top: 0;
+	right: 0;
+	left: 0;
+	padding: 25px;
+	visibility: hidden;
+	background: rgba(0, 0, 0, 0.8);
+	opacity: 0;
+	transition: opacity 0.3s ease-in-out;
+}
+
+.movie-thumb-link:hover .movie-thumb-text {
+	opacity: 1;
+	visibility: visible;
+	cursor: pointer;
+}
+
+.thumb-title {
+	font-size: 25px;
+	margin: 0;
+	width: 100%;
+}
+
+.release-date {
+	font-size: 18px;
+	margin: 0;
+}
+
+/* Movie Details */
+
+.movie-details-page {
+	padding: 60px;
+	background-size: cover;
+	background-position: center;
+	height: 100%;
+	min-height: 100vh;
+}
+
+.back-arrow-wrapper {
+	display: flex;
+	align-items: center;
+}
+
+.back-arrow-icon {
+	margin-right: 20px;
+	width: 30px;
+	height: 30px;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+.back-arrow-text {
+	color: white;
+	font-size: 20px;
+}
+
+.movie-details-card {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	width: 100%;
+	color: white;
+}
+
+.movie-poster {
+	width: 100%;
+	border: 2px solid white;
+	margin: 25px 0;
+}
+
+.movie-details-text {
+	background: rgba(0, 0, 0, 0.6);
+	padding: 25px;
+	font-size: 16px;
+}
+
+.movie-vote {
+	color: red;
+	align-self: flex-start;
+	margin-top: 0;
+}
+
+@media (min-width: 768px) {
+	.movie-list-container {
+		display: flex;
+		flex-direction: row;
+		flex-wrap: wrap;
```

Comment:
Very good job on the details for the media queries! They are often forgotten, nice to see that you put some hard work into this ! Remember you do not have to repeat something that is already mentioned for mobile version, e.j display: flex . Anything that is changed is the need to overwrite it for the next media query. Saves many lines of code 👌